src/include/sof/lib/wait.h

```diff
@@ -18,6 +18,7 @@
 #include <sof/schedule/task.h>
 #include <sof/trace/trace.h>
 #include <user/trace.h>
+#include <sof/platform.h>
```

Comment:
Should be after `#include <sof/drivers/timer.h>`.

Reply:
Done